src/main/java/edu/cornell/kfs/pdp/batch/service/impl/CuExtractPaymentServiceImpl.java

```diff
@@ -61,33 +62,31 @@
     */
     @Override
     public void extractAchPayments() {
-        LOG.debug("AchBundlerExtractPaymentServiceImpl MOD - extractAchPayments() started");
+        LOG.debug("CuExtractPaymentServiceImpl MOD - extractAchPayments() started");
```

Comment:
The logger will include the pull Class name in the log entry, so I don't think we need to include the class name in the log entry, just the function should do